src/app/components/header/header.component.html

```diff
@@ -81,28 +81,45 @@
                 <span class="header__label">UDC Deposit</span>
             </div>
             <ng-container *ngIf="tokens$ | async as tokens">
-                <ng-container
-                    *ngFor="
-                        let token of tokens
-                            | slice: 0:(tokens.length > 2 ? 1 : 2);
-                        trackBy: tokenTrackByFn
-                    "
-                >
+                <ng-container *ngIf="tokens[0] as token">
                     <div class="divider"></div>
                     <div
                         fxLayout="column"
                         fxLayoutGap="4px"
                         class="header__token-balance"
                     >
-                        <app-balance-with-symbol
-                            class="header__balance"
-                            [balance]="token.balance"
-                            [token]="token"
-                        ></app-balance-with-symbol>
+                        <span fxLayout="row" fxLayoutGap="6px">
+                            <app-balance-with-symbol
+                                class="header__balance"
+                                [balance]="token.balance"
+                                [token]="token"
+                            ></app-balance-with-symbol>
+                            <button
+                                *ngIf="(network$ | async)?.chainId !== 1"
```

Comment:
I would consider to give this expression a helpful name that reads better. E.g. `ngIf="!connectedToMainnet"` What do you think?

Reply:
Yes, makes sense as it also was like this before :)